File: Deberes/ExamenIIBimestre/app/src/main/java/com/example/exameniibimestre/Jugador.kt
package com.example.exameniibimestre

class Jugador (
    var id:Int,
    var numeroCamiseta:Int,
    var nombreCamiseta:String,
    var nombreCompletoJugador:String,
    var poderEspecialDos:String,
    var fechaIngresoEquipo:String,
    var goles:Int,
    var equipoFutbolId:Equipo,
    var longitud:String,
    var latitud:String
){

    override fun toString(): String {
        return this.nombreCompletoJugador
    }

}

File: 05_backend/backend-movil/api/models/Jugador.js
/**
 * Jugador.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {
    

    numeroCamiseta: {
      type: 'number',
      required: true,
    },
    nombreCamiseta: {
      type: 'string',
      required: true,
      minLength: 3,
      maxLength: 60,
    },
    nombreCompletoJugador: {
      type: 'string',
      required: true,
      minLength: 3,
      maxLength: 100,
    },
    poderEspecialDos: {
      type: 'string',
      required: true,
      minLength: 3,
      maxLength: 60,
    },
    fechaIngresoEquipo: {
      type: 'string',
      required: true,
    },
    goles: {
      type: 'number',
      required: true,
    },
    longitud: {
      type: 'string',
    },
    latitud: {
      type: 'string',
    },
    equipoFutbolId:{
      model:'equipo',
      required:true
    }
  },

};

